pkg/pit/metallb.go

```diff
@@ -66,9 +73,10 @@ func WriteMetalLBConfigMap(path string, v *viper.Viper, networks map[string]*csi
 	}
 	var configStruct MetalLBConfigMap
 	configStruct.Networks = make(map[string]string)
-	configStruct.ASN = v.GetString("bgp-asn")
 
```

Comment:
Should this be deleted?

Reply:
yes, that was an variable that used the same ASN for both "peer-asn" and "my-asn" and for all peers.   Now it sets the ASN separately for each peer.